simul/simul.go

```diff
@@ -260,3 +304,32 @@ func getStartStop(rcs int) (int, int) {
 	dbg.Lvl2("Range is", start, ":", stop)
 	return start, stop
 }
+
+// getRunWait returns either the command-line value or the value from the runconfig
+// file
+func getRunWait(rc platform.RunConfig) int {
```

Comment:
OK - here I also 'abuse' it to set a default variable. In the case of using `rc.GetInt`, we'd have to define the default-values at another place, too.